### PR TITLE
Mdformat tables

### PR DESCRIPTION
**Pull request by danlester**
Add mdformat-tables because tables were ruined without it.
Run mdformat on all docs so existing docs are formatted in accordance with the new rules.

## Types of changes

What types of changes does your PR introduce?

_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [x] Build related changes
- [ ] Other (please describe):

### Requires testing

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/source/user_guide/idle_culler.md [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
+To help jupyterhub-idle-culler cull user servers, we configure the kernel manager to cull idle kernels that would otherwise make the user servers report themselves as active which
+is part of what jupyterhub-idle-culler considers.
 
-***
-The expected behavior is that the server will be shut down and removed from the Qhub namespace once all Terminals and Kernels are considered idle or terminated, as well as any remaning connection is closed.
-***
+______________________________________________________________________
+
+The expected behavior is that the server will be shut down and removed from the Qhub namespace once all Terminals and Kernels are considered idle or terminated, as well as any
+remaning connection is closed.
```
Comment (viniciusdc) [inline docs/source/user_guide/idle_culler.md:15]:
```suggestion
The expected behavior is that the server will be shut down and removed from the Qhub namespace once all Terminals and Kernels are considered idle or terminated, as well as any remaining connections are closed.
```

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
-name: "Documentation \U0001F516"
-about: Issues and feedback related to the docs.
-title: "[DOC] - <title>"
-labels: ["type: documentation"]
+______________________________________________________________________
 
----
+name: "Documentation \\U0001F516" about: Issues and feedback related to the docs. title: "\[DOC\] - <title>" labels: \["type: documentation"\]
+
+______________________________________________________________________
```
Comment (viniciusdc) [inline .github/ISSUE_TEMPLATE/documentation.md:5]: This might need to be refactored again, not sure if Github will accept as a valid template

**Review by viniciusdc — COMMENTED** [review]
Thanks, Dan for running the mdformat in the docs and caching the issue with the tables. We should only remove the .github files from the path for now as it might break the templates.

File: .github/ISSUE_TEMPLATE/documentation.md (hunk 2 above)
Comment (danlester) [reply .github/ISSUE_TEMPLATE/documentation.md]: Good spot, I will restrict to docs only for now, and revert to earlier files elsewhere.

**Pull request merged**